### DISCUSSION TITLE
How do I remove a member's timeout?

### DISCUSSION THREAD

**Discussion by XInTheDark** [Rapptz/discord.py#9555] [Q&A] [answered]
I understand that `member.timeout()` times out a member, but there doesn't seem to be a function in the API reference that _removes_ this timeout. e.g. as a human, I have this option:
<img width="208" alt="image" src="https://github.com/Rapptz/discord.py/assets/61100393/193e1cc3-fbf9-4db2-af3b-cd07f9302ad9">

Advice on how to do this is much appreciated!

**Comment by LostLuma** [comment, marked as answer]
You can pass `None` as a timeout value:  

> If None is passed then the timeout is removed.

https://discordpy.readthedocs.io/en/latest/api.html#discord.Member.timeout

**Reply by Akshsays** [reply]
Thanks mate
